### DISCUSSION TITLE
PolyLookup: Dropdown values disappear when FetchXML passed in Lookup View

### DISCUSSION THREAD

**Discussion by shaharyar-z-ali** [khoait/DCE.PCF#172] [Q&A]
Not sure if I'm doing it correctly.

I'm passing a FetchXML in the lookup view box to make search enabled on name rather than the primary name column. When I'm doing this, the dropdown list becomes empty and nothing is shown when I search. 

Please help if I'm doing something wrong ?

FetchXML:
`<fetch>
  <entity name="crcbd_EmployerContactRepository">
    <attribute name="crcbd_ContactID"/>
    <attribute name="crcbd_Name"/>
    <attribute name="crcbd_ContactNumber"/>
    <attribute name="crcbd_ContactPosition"/>
    <attribute name="crcbd_Email"/>
    <order attribute="crcbd_ContactID" descending="false"/>
    <filter type="and">
      <condition attribute="statecode" operator="eq" value="0"/>		
      <condition attribute="crcbd_Name" operator="like" value="%{{PolyLookupSearch}}%"/>
    </filter>
  </entity>
</fetch>`

Screenshot component:
![image](https://github.com/user-attachments/assets/2ffd517f-92c7-44b1-b185-b482f6bb9326)

Screenshot output:
![image](https://github.com/user-attachments/assets/bd4f8b68-2a6b-48d4-8ce0-c27837502e09)

**Comment by khoait** [comment]
@shaharyar-z-ali could you try again with the new release v1.1.1.0 and let me know?

**Comment by nnarvekar** [comment]
@khoait I have FetchXML passed to filter the dropdown values. It works sometimes and shows "No options found" other times. If I refresh the screen, it shows the values in the dropdown. Any ideas on why it doesn't work sometimes? Thanks!

**Reply by khoait** [reply]
Hi @nnarvekar are you using the latest version? It would help if you can share your fetch and other properties, and screenshots as well. Thanks!

**Reply by nnarvekar** [reply]
Hi @khoait, I am using version1.1.1.0

Here's a screenshot of my fetchXML
![image](https://github.com/user-attachments/assets/ec6adc37-bba4-43bc-8828-0c21a14552d2)

And a screenshot of the other properties
![image](https://github.com/user-attachments/assets/ae1b4c71-0b59-49a1-b14e-4fd47c0b835a)

Screenshot of the field on my form
![image](https://github.com/user-attachments/assets/8d567e5d-260b-4624-a596-2d8ef0df2d59)

Thanks much

**Reply by khoait** [reply]
@nnarvekar can you check the `sec_exam.id`? is it null? can you test the fetchxml in Xrmtoolbox to see if it's returning data? also check in browser control if there is any error?

**Reply by nnarvekar** [reply]
Hello @khoait sec_exam.id is not null. 
It works fine when I test the fetchxml in Xrmtoolbox. 
I dont see any error in the browser. 
I see the issue where the drop down does not have any options on the new form sometimes. I have the sec_exam field on the form. I wonder if the fetchxml is running before the sec_exam field has fully loaded.
